cli/src/main/kotlin/io/spine/protodata/cli/Main.kt

```diff
@@ -234,18 +265,18 @@ internal class Run(version: String) : CliktCommand(
     private fun createRegistry(): ExtensionRegistry {
         val optionsProviders = loadOptions()
         val registry = ExtensionRegistry.newInstance()
-        optionsProviders.forEach { it.dumpTo(registry) }
+        optionsProviders.forEach { it.registerAt(registry) }
```

Comment:
I would say `registerIn`. 

"Register at" would rather point to the place where the action happens, not to the thing in which we register.  See [this conversation](https://ell.stackexchange.com/a/171467) on the subject.